### ISSUE TITLE
Nameserver settings are not saved

### ISSUE THREAD

**Issue by princeofnaxos** [Froxlor/Froxlor#639]
When editing a domain, the nameserver settings are not saved.

**Comment by d00p** [comment]
Please specify the used version....

**Comment by princeofnaxos** [comment]
Sorry... 0.10.0/master

**Comment by d00p** [comment]
a bit more info please...e.g. do you have nameserver usage enabled systemwide?

**Comment by princeofnaxos** [comment]
Yes, nameserver is enabled systemwide and works. I can also create new domains and activate the nameserver there. 

I had a few domains with own zonefile because in the old syscp, nameserver support was very poor. Now in Froxlor I try to get rid of these zonefiles but it wont accept the change.


Edwin Boersma
Fasterminds
http://www.fasterminds.com

Tel.: +46709726431
Tel.: +46313011801
